triage-party/base/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: triage-party
+spec:
+  selector:
+    matchLabels:
+      app: triage-party
+  template:
+    metadata:
+      labels:
+        app: triage-party
+    spec:
+      containers:
+        - name: triage-party
+          image: triage-party:latest
+          env:
+            - name: GITHUB_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: triage-party-github-token
+                  key: token
+            - name: PERSIST_PATH
+              value: "/app/pcache"
+            - name: PERSIST_BACKEND
+              value: "disk"
+          volumeMounts:
+            - name: config
+              mountPath: /app/config
+            - name: pcahce
+              mountPath: /app/pcache
+      volumes:
+        - name: config
+          configMap:
+            name: triage-party-config
+        - name: pcahce
```

Comment:
pcache*

Reply:
thanks for the catch, pushing a fixed version